recipes/manormfast/meta.yaml

```diff
@@ -1,6 +1,6 @@
 package:
   name: manormfast
-  version: "0.1.2"
+  version: "0.1.3"
```

Comment:
Note: I didn't forget matplotlib in the pypi package, just in this file. I don't know what other number there is to change.